tests/robustness/watch.go

```diff
@@ -143,3 +143,27 @@ external:
 		t.Errorf("Progress notify does not match, expect: %v, got: %v", expectProgressNotify, gotProgressNotify)
 	}
 }
+
+func validateWatchSequential(t *testing.T, reports []report.ClientReport) {
+	for _, r := range reports {
```

Comment:
What about 2 concurrent client watch requests to the same server? I think we need to process in different order than reports. I think we should only consider watch requests if they don't intersect.

Reply:
The solution came to my mind is to create a list of all `WatchResponse` that got sorted by Time.

Reply:
I would recommend to collect (per member) pairs (last revision, time) from watch responses and sort them by time. Then go through all watch requests and search the (last revision, time) to find the last one before sending the request. First event revision should be always greater or equal one we found.